packages/client/modules/teamDashboard/components/AgendaList/AgendaList.tsx

```diff
@@ -16,16 +16,14 @@ import AgendaItem from '../AgendaItem/AgendaItem'
 import AgendaListEmptyState from './AgendaListEmptyState'
 import {AgendaList_agendaItems} from '~/__generated__/AgendaList_agendaItems.graphql'
 
-const AgendaListRoot = styled('div')({
+const AgendaListRoot = styled('div')<{isMeeting: boolean}>(({isMeeting}) => ({
   display: 'flex',
   flexDirection: 'column',
-  // react-beautiful-dnd supports scrolling on 1 parent
-  // this is where we need it, in order to scroll a long list
-  overflow: 'auto',
+  overflow: isMeeting ? undefined : 'auto',
```

Comment:
I noticed that if there are many agenda items, there could be two vertical scrollbars. Now that the sidebar is scrollable in the meeting, we can remove this overflow